### ISSUE TITLE
create new keyboard shortcut for "first 150 base pairs (f1)" error

### ISSUE THREAD

**Issue by msingerb** [macarthur-lab/variant-curation-portal#228]
Can we create a new keyboard shortcut for the "first 150  base pairs (shortcut = f1)" eror? this offen just makes it list the curates as LoF (Shortcut = 1). We can probably change it to "fo" instead

**Comment by msingerb** [comment]
so this is still a problem. I think its because using "f" key makes the window open and close. can we change it to "hf"?

**Comment by nawatts** [comment]
I've only been able to reproduce this by either holding down "f" or waiting a few seconds between pressing "f" and "1".

> so this is still a problem. I think its because using "f" key makes the window open and close.

What browser / operating system are you using? This makes me wonder if there's some deeper issue related to the library we're using for keyboard shortcuts. We changed the shortcut for showing/hiding the form from "f" to "ff" to avoid this issue.

Also, the keyboard shortcut for "pext < 50% max" ("p5") may also have a similar issue since "5" is the shortcut for "Not LoF".
